docs/logger-mdc-instrumentation.md

```diff
@@ -19,21 +19,27 @@ event's MDC copy:
   (same as `Span.current().getSpanContext().getTraceFlags().asHex()`).
 
 Those three pieces of information can be included in log statements produced by the logging library
-by specifying them in the pattern/format.
+by specifying them in the pattern/format. This way any services or tools that parse the application 
+logs can correlate traces/spans with log statements.
 
-Tip: for Spring Boot configuration which uses logback, you can add MDC to log lines by overriding only the `logging.pattern.level`:
+> Note: If the current `Span` is invalid, there will be no OTEL info in the MDC
 
-```properties
-logging.pattern.level = trace_id=%mdc{trace_id} span_id=%mdc{span_id} trace_flags=%mdc{trace_flags} %5p
-```
+## Supported logging libraries
 
-This way any services or tools that parse the application logs can correlate traces/spans with log
-statements.
+> Note: There are also log appenders for exporting logs to OpenTelemetry, not to be confused with the MDC appenders
 
-## Supported logging libraries
+| Library | Version | Documentation                                                                        |
```

Comment:
WDYT about using the same column names as in https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#libraries--frameworks ?
```suggestion
| Library | Auto-instrumented versions | Standalone Library Instrumentation                                                                        |
```